server/data/migrations/20190301162501_billing.js

```diff
@@ -1,47 +1,11 @@
-exports.up = function(knex, Promise) {
-  return knex.schema.createTable("billing", billing => {
-    billing.increments("id");
-    billing.string("street", 255).notNullable();
-    billing.string("city", 128).notNullable();
-    billing.string("state", 128).notNullable();
-
-    billing.string("country", 255).notNullable();
-    billing.integer("zipcode").notNullable();
-    billing.string("mailingStreet", 255).notNullable();
-    billing.string("mailingCity", 128);
-    billing.string("mailingState", 128);
-    billing.string("mailingCountry", 255);
-    billing.integer("mailingZipcode").notNullable();
-    billing
-      .string("ccn")
-      .notNullable()
-      .unsigned();
-    billing
-      .integer("expireMonth")
-      .notNullable()
-      .unsigned();
-    billing
-      .integer("expireYear")
-      .notNullable()
-      .unsigned();
-    billing
-      .integer("code")
-      .notNullable()
-      .unsigned();
-    billing
-      .integer("monthlyBill")
-      .notNullable()
-      .unsigned();
-    billing.string("accountStatus", 128).notNullable();
-    billing.integer("targetsUsed").unsigned();
-    
-    billing.integer("accountType").unsigned();
-    billing.string("mobilePhone", 128);
-    billing.string("homePhone", 128);
-    billing.integer('users_id').unsigned();
+exports.up = function (knex, Promise) {
+  return knex.schema.createTable ('billing', billing => {
```

Comment:
This change was to satisfy the importing of stripe into our database. The response from the stripe is just a token but we manage to make it create a new response to give us what we need to identify the user as a pro, new, or investor account.